crates/cli/src/args.rs

```diff
@@ -366,3 +371,69 @@ pub struct Load {
     pub collections: usize,
     pub items: usize,
 }
+
+#[derive(Debug, Copy, Clone)]
+pub enum FormatWrapper {
+    Json(bool),
+    Ndjson,
+    Geoparquet(Option<Compression>),
+}
+
+impl ValueEnum for FormatWrapper {
+    fn value_variants<'a>() -> &'a [Self] {
+        &[
+            FormatWrapper::Json(true),
+            FormatWrapper::Json(false),
+            FormatWrapper::Ndjson,
+            FormatWrapper::Geoparquet(None),
+            FormatWrapper::Geoparquet(Some(Compression::LZ4)),
+            FormatWrapper::Geoparquet(Some(Compression::SNAPPY)),
+            // Do not know how to deal with options below:
+            // FormatWrapper::Geoparquet(Some(Compression::BROTLI(???))),
+            // FormatWrapper::Geoparquet(Some(Compression::from_str("BROTLI").unwrap())),
+        ]
```

Comment:
This is another way. Also includes listing all possible options with additional complications - I have no idea how to construct `FormatWrapper::Geoparquet(Compression::BROTLI(BrotliLevel)`

Reply:
I think you could just make BROTLI go to `"geoparquet[brotli(n)]"`, as the levels are all `u32`: https://docs.rs/parquet/latest/src/parquet/basic.rs.html#410-424